zephyr/module.yml

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+build:
+  kconfig: Kconfig
+  cmake: .
+  settings:
+    board_root: .
+    dts_root: .
```

As this is an example repo, then some comments to assist the reader would be nice.

```suggestion
  # Path to the Kconfig file that will be sourced into Zephyr Kconfig tree under
  # `Zephyr->Modules->example-application` 
  # Path is relative from root of this repository.
  kconfig: Kconfig

  # Path to the folder that contains the CMakeLists.txt file
  # to be included by Zephyr build system.
  # The `.` is the root of this repository
  cmake: .
  settings:
    # Additional roots for boards and DTS files.
    # The `.` is the root of this repository
    # Zephyr will use the `<board_root>/boards` for additional boards.
    board_root: .
    # Zephyr will use the `<dts_root>/dts` for additional dts files.
    # Zephyr will use the `<dts_root>/dts/bindings` for dts binding files.
    dts_root: .
```

---

I agree with adding comments in this file.  Let's also get the `module.yml` documented in a Zephyr guide and reference that too.

---

done